docs/api.rst

```diff
@@ -25,6 +25,17 @@ Types
 
 .. todo:: Add links to the upstream documentation for the GA4GH types.
 
+
```

Comment:
@jeromekelleher quick clarification: do you mean the todo section or the following BioMetadata section?